pages/services/kafka-zookeeper/2.2.0-3.4.11/install/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: layout.pug
+navigationTitle: 
+excerpt:
+title: Install and Customize
+menuWeight: 20
+
+model: /services/kafka-zookeeper/data.yml
+render: mustache
+---
+
+<!-- Imported from git@github.com:mesosphere/dcos-zookeeper.git:update-docs -->
+
+The default DC/OS Apache ZooKeeper Service installation provides reasonable defaults for trying out the service, but may not be sufficient for production use. You may require different configurations depeneding on the context of the deployment.
```

Comment:
Would it make sense to use the include files here: `#include /services/include/install1.tmpl
`?